unsupported/juno/hermes/build.rs

```diff
@@ -8,8 +8,15 @@
 use std::env;
 
 fn main() {
-    let hermes_build = env::var("HERMES_BUILD")
-        .expect("HERMES_BUILD must point to a Hermes CMake build directory");
+    let hermes_build = env::var("HERMES_BUILD").unwrap_or_else(|_| {
+        env::current_dir()
+            .unwrap()
+            .as_path()
+            .join("../../../build")
```

Comment:
The default value seems pretty arbitrary, for example it wouldn't work on my system since my build directory is in a completely different place. Is there any reason you can't just set the environment variable?

TBH, we haven't completely figured out how to integrate the two builds yet. The current situation is more of a hack, to get us started.

Reply:
The main reason was that without this change, rust-analyzer in VSCode fails to build the crate, meaning there was no autocomplete and so on. (I didn't investigate whether you can set env vars that are applied to rust-analyzer's cargo invocations). I can of course remove this again during the rebasing.

> The current situation is more of a hack, to get us started.

(Another part of this is probably compiling the C++ side with the same targets that are specified by Cargo, to create a wasm build of this transform.)

Reply:
We addressed that in https://github.com/facebook/hermes/commit/c5c0ce04c8c0bfd0e3348df65b21ba85da54567b .